<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleDock</class>
 <widget class="QDockWidget" name="ConsoleDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="terminal">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="windowTitle">
   <string>Messages</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="autoFormatting">
       <set>QTextEdit::AutoAll</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="searchEdit">
      <property name="placeholderText">
       <string>Quick search</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="clearButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="icon">
       <iconset theme="edit-clear">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="saveButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="icon">
       <iconset theme="document-save">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>searchEdit</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
